c2cgeoportal/scaffolds/update/+package+/static/mobile/app/plugin/StatefulMap.js

```diff
@@ -117,6 +123,9 @@ Ext.define('App.plugin.StatefulMap', {
         if (q.map_x && q.map_y) {
             state.lonlat = [ q.map_x, q.map_y ];
             state.zoom = q.map_zoom;
+            if (parseInt(q.map_marker) === 1) {
```

Comment:
Thanks for the review, I will rename `map_marker` to `map_crosshair` and try to simplify the code as suggested !